pkg/scaling/resolver/aws_secretManager_handler.go

```diff
@@ -0,0 +1,116 @@
+package resolver
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/secretsmanager"
```

Comment:
We have to use aws-sdk-go-v2. 
https://pkg.go.dev/github.com/aws/aws-sdk-go-v2/service/secretsmanager

I'll review this file once it's updated :)

Reply:
Done.